CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+# The Rust Code of Conduct
+
+The Code of Conduct for this repository [can be found online](https://www.rust-lang.org/conduct.html).
```

Comment:
I like the idea of adding a link to the code of conduct. What about phrasing it like:

> This repository follows the Rust Project Code of Conduct, which [can be found on the Rust Language web site](https://www.rust-lang.org/conduct.html).